### PR TITLE
ENT-4579: overriding chunk size in SAP and Canvas Integrations 

### PR DESCRIPTION
**Pull request by kiram15**
Overrides default transmission chunk size from 500 to 1 for SAP SuccessFactors enterprise customer configuration after learning that the large number can actually prevent course items from being transmitted at all. Now mimics behavior seen in LMS configurations. 

PR corresponds to [ENT-4579](https://openedx.atlassian.net/browse/ENT-4579) and [ENT-1372](https://openedx.atlassian.net/browse/ENT-1372). 

**Merge checklist:**
- [x] Any new requirements are in the right place (do **not** manually modify the `requirements/*.txt` files)
    - `base.in` if needed in production but edx-platform doesn't install it
    - `test-master.in` if edx-platform pins it, with a matching version
    - `make upgrade && make requirements` have been run to regenerate requirements
- [x] `make static` has been run to update webpack bundling if any static content was updated
- [x] `./manage.py makemigrations` has been run
    - Checkout the [Database Migration](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations) Confluence page for helpful tips on creating migrations.
    - *Note*: This **must** be run if you modified any models.
      - It may or may not make a migration depending on exactly what you modified, but it should still be run.
    - This should be run from either a venv with all the lms/edx-enterprise requirements installed or if you checked out edx-enterprise into the src directory used by lms, you can run this command through an lms shell.
        - It would be `./manage.py lms makemigrations` in the shell.
- [x] [Version](https://github.com/edx/edx-enterprise/blob/master/enterprise/__init__.py) bumped
- [x] [Changelog](https://github.com/edx/edx-enterprise/blob/master/CHANGELOG.rst) record added
- [x] Translations updated (see docs/internationalization.rst but also this isn't blocking for merge atm)

**Post merge:**
- [ ] Tag pushed and a new [version](https://github.com/edx/edx-enterprise/releases) released
    - *Note*: Assets will be added automatically. You just need to provide a tag (should match your version number) and title and description.
- [ ] After versioned build finishes in [GitHub Actions](https://github.com/edx/edx-enterprise/actions), verify version has been pushed to [PyPI](https://pypi.org/project/edx-enterprise/)
    - Each step in the release build has a condition flag that checks if the rest of the steps are done and if so will deploy to PyPi.
    (so basically once your build finishes, after maybe a minute you should see the new version in PyPi automatically (on refresh))
- [ ] PR created in [edx-platform](https://github.com/edx/edx-platform) to upgrade dependencies (including edx-enterprise)
    - This **must** be done after the version is visible in PyPi as `make upgrade` in edx-platform will look for the latest version in PyPi.
    - Note: the edx-enterprise constraint in edx-platform **must** also be bumped to the latest version in PyPi.

### CODE REVIEW

**Review by alex-sheehan-edx — APPROVED** [review]
Assuming you've tested locally, LGTM 👍

**Review by taliaedX — COMMENTED** [review]
It looks good. There is one change that I mention that will fix the test currently failing and I think this might be the point where we get you setup locally with integrated channels also since the rest of the sprint is also integrated channels. There is an office hours on Friday apparently, but anyone on the team (everybody except Lael worked on integrated channels 😂 ) can walk you through setup earlier if you'd prefer.

File: integrated_channels/cornerstone/migrations/0007_auto_20210707_1258.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cornerstone', '0006_auto_20191001_0742'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='cornerstoneenterprisecustomerconfiguration',
```
Comment (taliaedX) [inline integrated_channels/cornerstone/migrations/0007_auto_20210707_1258.py:14]: Oooo no, no no let's not do this.

**Review by taliaedX — CHANGES_REQUESTED** [review]
I am blocking this PR to prevent this from even accidentally merging.

There are some unknowns in the behaviour modification of Degreed and Cornerstone and any others that aren't explicitly called out in the tickets. Even when called out in the tickets, we want to test to verify they all work. That's a lot of integrations to verify and for no discernible reason. Cornerstone is particularly uncertain to me. Let's just limit this to the scope of the tickets for now unless we explicitly are told the other integrations are also failing on large chunks.

File: integrated_channels/integrated_channel/models.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ class EnterpriseCustomerPluginConfiguration(TimeStampedModel):
     )
 
     transmission_chunk_size = models.IntegerField(
-        default=500,
+        default=1,
```
[the remark this answers is not in the document]
Comment (taliaedX) [reply integrated_channels/integrated_channel/models.py:56]: ie delete the `transmission_chunk_size` definitions from the individual models.
And for Degreed & Cornerstone and any others I didn't catch in this batch of commits, add `transmission_chunk_size = models.IntegerField(default=500)` to the enterprise customer configuration.

[the remark this answers is not in the document]
Comment (kiram15) [reply integrated_channels/integrated_channel/models.py:56]: I made these changes but thought the same thing, so I reached out to Rob Smith and he agreed with you so I'll revert it!

**Review by taliaedX — APPROVED** [review]

**Comment by taliaedX** [comment]
Approved now. Don't forget to squash some more!

**Pull request merged**